README.rdoc

```diff
@@ -173,6 +173,28 @@ Or how about persisting the session...
       end
   end
 
+
+== CSRF Protection
+
+Because Authlogic introduces its own methods for storing user sessions, the CSRF (Cross Site Request Forgery) protection that is built into Rails will not work out of the box.
```

Comment:
How about adding this sentence to the above: "No generally applicable mitigation by the authlogic library is possible, because the instance variable you use to store a reference to the user session in `def current_user_session` will not be known to authlogic." 
Open to other suggestions.